ovos-i2csound

```diff
@@ -73,10 +73,11 @@ compare_kernel_version() {
 main() {
     # Create a variable to hold the detected device name
     local i2c_device_name
+    local i2c_device_file=$HOME/.config/mycroft/i2c_platform
```

Comment:
Instead of `mycroft`, consider `${OVOS_CONFIG_BASE_FOLDER:-mycroft}` to match ovos-config directory handling

Reply:
this does not use a config nor is it part of ovos-core, IMHO it should use a static directory, this is not meant to be configurable, it is the output of a standalone program, anything using ovos-i2csound should follow the docs and check the static path, a layer of configuration here doesnt make sense to me and only introduces complexity and surface area for bugs